webhook_to_fedora_messaging_messages/forgejo/v1.py

```diff
@@ -0,0 +1,102 @@
+# SPDX-FileCopyrightText: 2024-2025 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+import json
+import typing
+from hashlib import sha1, sha256
+
+from ..base import Webhook2FedMsgBase
+from .utils import summarize_repository_event
+
+
+class ForgejoMessageV1(Webhook2FedMsgBase):
+    @property
+    def app_name(self) -> str:
+        """Application name on Fedora Messaging"""
+        return "Forgejo"
+
+    @property
+    def target_id(self) -> str:
+        """Webhook identifier specific to Forgejo"""
+        return self.body["body"]["repository"]["id"]
+
+    @property
+    def target_type(self) -> str:
+        """Webhook type specific to Forgejo - Computed"""
+        return "repository"
```

Comment:
Could you explain this please? What do you mean by "Computed"? If it's always "repository", then why is it useful? Do you expect other target types in the future? Isn't there a field in the webhook body that could be used to be more future-proof rather than a static string?

Reply:
Going forward, I see this changing.

There is the `x-forgejo-event-type` header that goes into describe specifically what changes took place and then there is the more generic `x-forgejo-event` header that we make use of over here. Take, for instance, if it were for a "repository" installation type - the `x-forgejo-event-type` can have possible values of `assigned`, `labeled` etc. when the `x-forgejo-event` is set to `issues`. For the "namespaces" installation, this can be different and so on.

Reply:
Do you have a link to the docs about this? I haven't found anything relevant about installation types.

Reply:
Their documentation is super scarce (with a bunch of linkbacks to the Gitea docs). I could send you some logs that I captured from a makeshift webhook receiver that I wrote, from where I got to know this.

Reply:
I have DMed you the logs on Slack. In the interest of time, I am abandoning my efforts of futureproofing this and instead removing this property entirely. We might need to add this later but let us build the bridge when we get there.